### DISCUSSION TITLE
Numeric values

### DISCUSSION THREAD

**Discussion by jrbarnes9** [ONSdigital/design-system#279] [Feedback]
Use this place to discuss the [numeric values pattern](https://ons-design-system.netlify.com/patterns/numeric-values) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by saedwards** [comment]
@will-madebymoon Symbols for different numeric value types in the pattern library aren't clickable. Is this expected? When you click the field, the focus surrounds the symbol as well as the field. We could make this clickable, which would really help on mobiles & tablets.

**Comment by armstrongb** [comment]
I don't think that this has been confirmed as an issue.
Suggest that if we want to check if clicking on the 'symbols' is an issue for users that we gather metrics via GTM for clicks on the `<abbr>` element.

**Reply by jrbarnes9** [reply]
Issue #1680 raised

**Comment by jrbarnes9** [comment]
The description on an input with suffix/prefix prevents the `title` being read out on Mac VoiceOver

So the user does not hear the units required for the input.

**Reply by jrbarnes9** [reply]
Issue #1581 raised

**Comment by jrbarnes9** [comment]
We should add guidance around using unit prefix/suffixes to ensure uncommon unit abbreviations are spelled out in the question or input label.  Also, update all the examples to ensure the unit is spelt out in the label.

**Reply by jrbarnes9** [reply]
Issue #1604 raised

**Comment by jrbarnes9** [comment]
How can we improve financial numeric input to avoid surveys feeling like they need to put all this kind of instruction in the hint text?
<img width="606" alt="Screenshot 2021-12-16 at 12 00 20" src="https://user-images.githubusercontent.com/43346934/146368798-8e29972d-f638-4d74-bed5-61975b21db5b.png">

**Reply by armstrongb** [reply]
Part of the historical problem is that on paper these businesses were asked to enter `56` for `56,000` as the the `000` was pre-filled values on the form. On some forms that collect larger values `000,000` was printed meaning `12` was written for `£12,000,000`. 

The original solution for this was to ensure that all values were shown in full on the summary screen, including pence, to make clear the true value being submitted. That way anyone entering `56` as they had habitually entered rounded thousands would be shown `£56.00` and given an opportunity to correct it to `£56,000`.

**Comment by Al-Davies** [comment]
User Research findings from 27.9.22 (eQ team)

- Where users are asked to provide % breakdowns, they would expect an error, they would expect validation to kick in after pressing the save and continue button
- Users don't necessarily expect to be told how much they have entered i.e. 102%, they would expect to have to work that out themselves

**Reply by jrbarnes9** [reply]
<img width="274" alt="Screenshot 2022-10-07 at 11 32 15" src="https://user-images.githubusercontent.com/43346934/194533870-45973afe-c5ed-4382-88bd-4e0c8f75a221.png">

**Reply by jrbarnes9** [reply]
I think these findings should feed into the publication of a new pattern 'How to ask for... Percentages' as it's quite a specific case where you're asking the user to provide multiple answers to complete a total figure.

Would be worth exploring this further to see if there's anything else we could do to make the process simpler e.g. sliders, auto-filling the final percentage.
